aldy/resources/genes/cyp2c19.yml

```diff
@@ -10,597 +10,649 @@ alleles:
   CYP2C19*30.001:
     pharmvar: https://www.pharmvar.org/haplotype/102
     activity: uncertain function
     label: CYP2C19*30
     evidence: L
     mutations:
-    - [17426, C>T, rs145328984, R73C]
+      - [17426, C>T, rs145328984, R73C]
   CYP2C19*31.001:
     pharmvar: https://www.pharmvar.org/haplotype/103
     activity: uncertain function
     label: CYP2C19*31
     evidence: L
     mutations:
-    - [5124, C>T, rs17885098]
-    - [17441, C>T, rs1564660997, H78Y]
-    - [85186, A>G, rs3758581, I331V]
+      - [5124, C>T, rs17885098]
+      - [17441, C>T, rs1564660997, H78Y]
+      - [85186, A>G, rs3758581, I331V]
   CYP2C19*32.001:
     pharmvar: https://www.pharmvar.org/haplotype/99
     activity: uncertain function
     label: CYP2C19*32
     evidence: L
     mutations:
-    - [5124, C>T, rs17885098]
-    - [17505, A>G, rs1288601658, H99R]
-    - [85186, A>G, rs3758581, I331V]
+      - [5124, C>T, rs17885098]
+      - [17505, A>G, rs1288601658, H99R]
+      - [85186, A>G, rs3758581, I331V]
   CYP2C19*33.001:
     pharmvar: https://www.pharmvar.org/haplotype/101
     activity: uncertain function
     label: CYP2C19*33
     evidence: L
     mutations:
-    - [5124, C>T, rs17885098]
-    - [22899, G>A, rs370803989, D188N]
-    - [85186, A>G, rs3758581, I331V]
+      - [5124, C>T, rs17885098]
+      - [22899, G>A, rs370803989, D188N]
+      - [85186, A>G, rs3758581, I331V]
   CYP2C19*34.001:
     pharmvar: https://www.pharmvar.org/haplotype/91
     activity: uncertain function
     label: CYP2C19*34
     evidence: D
     mutations:
-    - [5013, G>A, rs367543001]
-    - [5032, C>T, rs367543002, P3S]
-    - [5035, T>C, rs367543003, F4L]
+      - [5013, G>A, rs367543001]
+      - [5032, C>T, rs367543002, P3S]
+      - [5035, T>C, rs367543003, F4L]
   CYP2C19*35.001:
     pharmvar: https://www.pharmvar.org/haplotype/93
     activity: no function
     label: CYP2C19*35
     evidence: D
     mutations:
-    - [4113, G>A, rs76822098]
-    - [5124, C>T, rs17885098]
-    - [17687, A>G, rs12769205, splice defect]
-    - [85186, A>G, rs3758581, I331V]
+      - [4113, G>A, rs76822098]
+      - [5124, C>T, rs17885098]
+      - [17687, A>G, rs12769205, splice defect]
+      - [85186, A>G, rs3758581, I331V]
   CYP2C19*35.002:
     pharmvar: https://www.pharmvar.org/haplotype/1540
     activity: no function
     evidence: D
     mutations:
-    - [5080, A>C, rs17882687, I19L]
-    - [5124, C>T, rs17885098]
-    - [17687, A>G, rs12769205, splice defect]
-    - [85186, A>G, rs3758581, I331V]
+      - [5080, A>C, rs17882687, I19L]
+      - [5124, C>T, rs17885098]
+      - [17687, A>G, rs12769205, splice defect]
+      - [85186, A>G, rs3758581, I331V]
   CYP2C19*36.001:
     activity: no function
     mutations:
-    - [CYP2C19, deletion]
+      - [CYP2C19, deletion]
     ignored: true
   CYP2C19*37.001:
     activity: no function
     mutations:
-    - [CYP2C19, "deletion:e2,i2,e3,i3,e4,i4,e5"]
+      - [CYP2C19, 'deletion:e1,i1,e2,i2,e3,i3,e4,i4,e5']
```

Comment:
This is the meat of the PR, other changes are all enforcing correct YAML formatting